.circleci/config.yml

```diff
@@ -1192,6 +1215,7 @@ workflows:
       - node-integration-14
       - node-integration-16
       - node-integration-latest
+      - node-upstream-node
```

Comment:
This should also run nightly. As mentioned above, it should be in a new nightly workflow, for example `nightly-upstream`.